<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_MainWindow</class>
 <widget class="QMainWindow" name="ui_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="grid_layout" rowstretch="10,90" columnstretch="20,80,0">
      <item row="1" column="0">
       <widget class="MobdecProjectOption" name="widget_mobdec_project_option" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_mobdec_project">
          <property name="text">
           <string>MoBDEc project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_box_mobdec_project_list"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tool_button_previous_frame">
          <property name="text">
           <string>|&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool_button_pause">
          <property name="text">
           <string>||</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_current_frame">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_slash">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_total_frame">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool_button_play">
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tool_button_next_frame">
          <property name="text">
           <string>&gt;|</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QGridLayout" name="grid_layout_view"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MobdecProjectOption</class>
   <extends>QWidget</extends>
   <header location="global">view/mobdecProjectOption.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
